Jenkinsfile

```diff
@@ -0,0 +1,85 @@
+pipeline {
+    options {
+        disableConcurrentBuilds()
+        timestamps()
+    }
+    agent {
+        node { label 'master' }
+    }
+    environment {
+        GITHUB_CREDENTIALS = credentials('brave-builds-github-token-for-pr-builder')
+        BC_REPO = "https://${GITHUB_CREDENTIALS}@github.com/brave/brave-browser"
+        BC_BRANCH = "brave-core-${GIT_BRANCH}"
+    }
+    stages {
+        stage('checkout') {
+            steps {
+                echo "BRANCH_NAME: ${BRANCH_NAME}"
```

Comment:
nitpick: since there are all the `echo` lines in Jenkins console (between each echo), it might be more readable if this was all one line... like:
```
echo "BRANCH_NAME: ${BRANCH_NAME}" && \
echo "CHANGE_TARGET: ${CHANGE_TARGET}" && \
...
```

Reply:
<img width="774" alt="screen shot 2019-01-14 at 10 07 40 am" src="https://user-images.githubusercontent.com/4733304/51128174-784ff880-17e4-11e9-9d2a-ca1dd3d4cbae.png">

Reply:
let me just take those out

Reply:
done, it's going to run the pipeline now